dotlottie-rs/benches/benchmarks.rs

```diff
@@ -6,7 +6,177 @@ const HEIGHT: u32 = 1000;
 
 fn load_animation_data_benchmark(c: &mut Criterion) {
     let player = DotLottiePlayer::new(Config::default());
-    let data = std::str::from_utf8(include_bytes!("../tests/fixtures/test.json")).unwrap();
+    let data = r#"
```

Comment:
imo It shouldn’t matter, data loading is outside the `bench_function` scope, which only records the perf